code/game/jobs/whitelist.dm

```diff
@@ -28,26 +28,15 @@ var/list/whitelist = list()
 	if (!text)
 		log_misc("Failed to load config/alienwhitelist.txt")
 	else
-		alien_whitelist = text2list(text, "\n")
+		alien_whitelist["human"] = TRUE
+		alien_whitelist["Human"] = TRUE
```

Comment:
It's so you don't have to write "Human" in the alien whitelist. I had "human" in there because the original code had it along with "Human", and I'm worried about breaking stuff.

Reply:
Have just "Human"